packages/vue-vanilla/src/cells/BooleanCell.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <input
+    :id="cell.id + '-input'"
+    type="checkbox"
+    :class="styles.control.input"
+    :checked="!!cell.data"
+    :disabled="!cell.enabled"
+    :autofocus="appliedOptions.focus"
+    :placeholder="appliedOptions.placeholder"
+    @change="onChange"
+    @focus="isFocused = true"
+    @blur="isFocused = false"
+  />
+</template>
+
+<script setup lang="ts">
+import type { CellProps, RankedTester } from '@jsonforms/core';
+import { isBooleanControl, rankWith } from '@jsonforms/core';
+import { useJsonFormsCell } from '@jsonforms/vue';
+import { useVanillaCell } from '../util';
+
+const props = defineProps<CellProps>();
+
+const input = useVanillaCell(
+  useJsonFormsCell(props),
+  (target) => target.checked
+);
+const { styles, cell, appliedOptions, onChange, isFocused } = input;
+
+defineOptions({
+  tester: rankWith(1, isBooleanControl) as RankedTester,
+});
```

Comment:
Minor: Instead of casting we should make sure that the type system checks for us, e.g.
```suggestion
const tester: RankedTester = rankWith(1, isBooleanControl);
defineOptions({
  tester,
});
```

Reply:
that is not possible with `defineOptions`

> `defineOptions()` in <script setup> cannot reference locally declared variables because it will be hoisted outside of the setup() function. If your component options require initialization in the module scope, use a separate normal <script> to export the options instead.

see https://vuejs.org/api/sfc-script-setup.html#defineoptions

an alternative could be:
```ts
defineOptions(
  ((): { tester: RankedTester } => {
    const tester: RankedTester = rankWith(1, isBooleanControl);
    return { tester };
  })()
);
```
but that looks a bit over engineered

Reply:
another option could be a second script tag without the "setup" key next to the first one:
```ts
<script setup lang="ts">
import { rendererProps, useJsonFormsEnumCell } from '@jsonforms/vue';
import type { ControlElement } from '@jsonforms/core';
import { useVanillaCell } from '../util';

const props = defineProps(rendererProps<ControlElement>());

const input = useVanillaCell(
  useJsonFormsCell(props),
  (target) => target.checked
);
const { styles, cell, appliedOptions, onChange, isFocused } = input;
</script>

<script lang=ts>
import { isBooleanControl, type RankedTester, rankWith } from '@jsonforms/core';
export const tester: RankedTester = rankWith(1, isBooleanControl);
</script>
```

but your es-lint dont like it and move the imports to the first script tag